app/models/miq_ui_worker.rb

```diff
@@ -20,4 +20,17 @@ def friendly_name
   end
 
   include MiqWebServerWorkerMixin
+  include MiqWorker::ServiceWorker
+
+  def self.supports_container?
+    true
+  end
+
+  def container_port
+    3001
+  end
+
+  def container_image_name
+    "manageiq/manageiq-ui-worker"
```

Comment:
Should you be able to change the org name?

Reply:
Can we drop the "manageiq-" from the container name?

Reply:
I want to put all of this in settings like we did for the awx stuff, but I think I'm going to change that in a follow-up.

As for the name, I don't really have a preference. Thoughts @Fryguy ?

Reply:
@carbonin and I discussed moving this into Settings, but I wanted to get this in first, and do that as an incremental improvement.